tests/data/fancyHelloWorld.html

```diff
@@ -1,31 +1,31 @@
-<!DOCTYPE html>
-<head>
-<meta charset="UTF-8">
-<style>
-.r1 {color: #8cc265; text-decoration-color: #8cc265}
-.r2 {color: #4aa5f0; text-decoration-color: #4aa5f0}
-.r3 {color: #e05561; text-decoration-color: #e05561}
-.r4 {background-color: #e05561}
-.r5 {background-color: #d18f52}
-.r6 {background-color: #8cc265}
-.r7 {font-style: italic; text-decoration: line-through}
-.r8 {font-weight: bold}
-.r9 {text-decoration: underline}
-.r10 {background-color: #42b3c2}
-.r11 {background-color: #4aa5f0}
-.r12 {background-color: #c162de}
-body {
-    color: #abb2bf;
-    background-color: #282c34;
-}
-</style>
-</head>
-<html>
-<body>
-    <code>
-        <pre style="font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace">👋<span class="r1">Hello</span>, <span class="r2">World</span>🌏<span class="r3">!</span>
-<span class="r4">👋</span><span class="r5">🦄</span><span class="r6">🐘</span><span class="r7">13</span><span class="r8">3</span><span class="r9">7</span><span class="r10">🍄</span><span class="r11">🎃</span><span class="r12">🐦</span>
-</pre>
-    </code>
-</body>
-</html>
+<!DOCTYPE html>
+<head>
+<meta charset="UTF-8">
+<style>
+.r1 {color: #8cc265; text-decoration-color: #8cc265}
+.r2 {color: #4aa5f0; text-decoration-color: #4aa5f0}
+.r3 {color: #e05561; text-decoration-color: #e05561}
+.r4 {background-color: #e05561}
+.r5 {background-color: #d18f52}
+.r6 {background-color: #8cc265}
+.r7 {font-style: italic; text-decoration: line-through}
+.r8 {font-weight: bold}
+.r9 {text-decoration: underline}
+.r10 {background-color: #42b3c2}
+.r11 {background-color: #4aa5f0}
+.r12 {background-color: #c162de}
+body {
+    color: #abb2bf;
+    background-color: #282c34;
+}
+</style>
+</head>
+<html>
+<body>
+    <code>
+        <pre style="font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace">👋<span class="r1">Hello</span>, <span class="r2">World</span>🌏<span class="r3">!</span>
+<span class="r4">👋</span><span class="r5">🦄</span><span class="r6">🐘</span><span class="r7">13</span><span class="r8">3</span><span class="r9">7</span><span class="r10">🍄</span><span class="r11">🎃</span><span class="r12">🐦</span>
+</pre>
+    </code>
+</body>
+</html>
```

Comment:
Do you mind sharing what OS you are on? Just curious as the rendered html is moderately different to that on WIndows

Reply:
Ah cheers :)